src/rpc/blockchain.cpp

```diff
@@ -539,10 +553,16 @@ static RPCHelpMan getblockheader()
     if (!request.params[1].isNull())
         fVerbose = request.params[1].get_bool();
 
+    int count = 1;
+    if (!request.params[2].isNull())
+        count = request.params[2].get_int();
```

Comment:
```suggestion
    const auto count{request.params[2].isNull() ? 1 : request.params[2].getInt<int>()};
```

can be written in a single line